2603

Lokinay opened the pull request.

Lokinay committed:
Create test_cases

Lokinay committed:
Create bug_report

Lokinay committed:
подправил табуляцию

Lokinay committed:
исправил интервал между строками для удобства чтения

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+	Шаги воспроизведения:
+		1. Открыть главную страницу сайта https://www.onetwotrip.com
+		2. Нажать на боковую кнопку "Поддержка"
+		3. В поле вопроса нажать на ссылку "Прикрепить файл"
```
"Присоединить файл"

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+		1. Открыть главную страницу сайта https://www.onetwotrip.com
+		2. Нажать на боковую кнопку "Поддержка"
+		3. В поле вопроса нажать на ссылку "Прикрепить файл"
+		4. Выбрать файл большого размера
```
Здесь нужно указать, что такое большой файл. Понятно, что для этого сайта ты ограничение не знаешь, но можно выдумать какое-нибудь Nmax. Иначе при прогоне кейса нет уверенности, что тестировщик, который по нему тестирует выберет файл, который уйдет за границы допустимого.

Lokinay committed:
исправил замечания

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+	Шаги воспроизведения:
+		1. Открыть главную страницу сайта https://www.onetwotrip.com
+		2. Нажать на боковую кнопку "Поддержка"
+		3. В поле вопроса,зажимая какую-либо клавишу,набрать большое количество символов
```
А здесь важно именно зажимать клавишу или можно ее много раз нажать? 
Лучше указывать конкретный способ ввода, когда он имеет значение. Если способ ввода не важен, но имеет значение ввод это или вставка, то можно ограничиться указанием "ввести с клавиатуры". Если и способ ввода не важен, то тоже это стоит указать (например, "ввести с клавиатуры или вставить в поле из буфера обмена")

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+№1
+  	Короткое описание:
+		Ограничение размера приклепляемого документа в поле вопроса
```
Так как это заголовок описания проблемы, то в нем стоит указывать, что именно не работает, в чем состоит проблема. Сейчас у тебя в заголовке описание функционала, который работает не так, как ожидалось, а вот баг это или доработка, или запрос на разработку - не ясно.
Лучше составлять название так <где>: <проблема>.  Например: Поддержка: отправляется файл больше Nmax+1

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+	
+№3
+  	Короткое описание:
+		Запрет на ввод спецсимволов и кода в поле "Тема сообщения"
```
Краткое описание нужно уточнить

aleksandra-jashina replied:
Стоит учтонить, что речь о JS-коде, а не о HTML или php, например

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+	
+№2
+  	Короткое описание:
+		Ограничение количества символов в поле вопроса
```
Тоже нужно уточнить краткое описание

aleksandra-jashina commented on 2603/test_cases:
```diff
@@ -0,0 +1,45 @@
+		3. Установить фокус на поле «Куда»
+		4. Ввести название города,в который полетим
+		5. Установить фокус на поле «Туда»
+		6. Выбрать корректную дату полета
```
Тут нужно либо кратко описать, что такое корректная дата, либо дать ссылку на описание в документации

aleksandra-jashina commented on 2603/test_cases:
```diff
@@ -0,0 +1,45 @@
+		7. Нажать на кнопку «Найти»
+	
+	Ожидаемый результат: 
+	Фактический результат: Результаты отображены корректно, есть возможность просмотра дополнительной информации
```
Как понять, например, новому тестировщику на проекте, что результаты отображены корректно? Как именно должна выглядеть возможность просмотра дополнительной информации?

aleksandra-jashina replied:
Почему в тест кейсе есть Ожидаемый и Фактический результат?

aleksandra-jashina commented on 2603/test_cases:
```diff
@@ -0,0 +1,45 @@
+		1. Открыта главная страница сайта https://www.onetwotrip.com
+		2. Активирована функция "Наборный"
+	Шаги:
+		1. Установить фокус на поле «Откуда» рейса №1
```
Тут наверняка есть правила, по котороым надо заполнять поля даты и все поля рейса 2, чтобы получить корректный результат, а не сообщение об ошибке, например

Lokinay committed:
Update test_cases

Исправил описание корректной даты,дополнительной информации,убрал фактический результат

Lokinay committed:
Update bug_report

Исправил все замечания

aleksandra-jashina commented on 2603/bug_report:
```diff
@@ -0,0 +1,69 @@
+	Шаги воспроизведения:
+		1. Открыть главную страницу сайта https://www.onetwotrip.com
+		2. Нажать на боковую кнопку "Поддержка"
+		3. В поле вопроса ввести с клавиатуры большое количество символов,не менее Nmax(см. документацию)
```
"Не менее Nmax" ведь допускает ввод Nmax символов. Будет ли считаться ошибкой, если отправится сообщение, содержащее Nmax символов?

Lokinay replied:
Не должно считаться.Если ограничение 100 символов,то эти 100 символов мы можем ввести.Да,здесь по-хорошему надо было добавить БОЛЬШЕ Nmax

Lokinay committed:
Update bug_report
